be2-scala/README.md

```diff
@@ -95,13 +95,13 @@ environment:
 ```sh
 # install sdkman
 curl -s "https://get.sdkman.io" | bash
-# install java zulu 11
-sdk install java 11.0.18-zulu
+# install java zulu 17
+sdk install java 17.0.8-zulu
```

Comment:
general question:
do we want to stick with the zulu distribution ?
https://sdkman.io/jdks

Reply:
There is this website: https://whichjdk.com

Given their opinion we should go preferably with Temurin but Zulu seems fine. As the website explains multiple companies are behind the Eclipse Foundation where Azul are the only one behind Zulu. Not depending on a single company is better so we can try switching to it, what do you think ?

Reply:
Agreed !